### ISSUE TITLE
No EBS driver error message

### ISSUE THREAD

**Issue by stobias123** [rancher/rancher#10752]
**Rancher versions:**
rancher/server: 1.6.7
rancher/agent:  1.2.5

**Infrastructure Stack versions:**
```
rancher/healthcheck:v0.3.1
rancher/storage-nfs:v0.8.5
rancher/storage-ebs:v0.8.1
rancher/net:v0.11.7
rancher/dns:v0.15.1
rancher/net:holder
rancher/metadata:v0.9.3
rancher/net:v0.11.7
rancher/network-manager:v0.7.7
rancher/agent:v1.2.5
```

**Docker version: (`docker version`,`docker info` preferred)**
root@ip-10-103-253-100:~# docker info
Containers: 14
 Running: 13
 Paused: 0
 Stopped: 1
Images: 83
Server Version: 1.13.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 318
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local rancher-ebs
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: aa8187dbd3b7ad67d8e5e3a15115d3eef43a7ed1
runc version: 9df8b306d01f59d3a8029be411de015b7304dd8f
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.4.0-1041-aws
Operating System: Ubuntu 16.04.3 LTS
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 1.952 GiB
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**
```
NAME="Ubuntu"
VERSION="16.04.3 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.3 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial
4.4.0-1041-aws
```
**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**
AWS
**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
HA Rancher, RDS db
**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**
cattle

**Steps to Reproduce:**
use an m5 instance with the above listed ebs driver to create a stack
**Results:**
Your stack with an EBS volume will never successfully start. 
There is never any sort of error message or retry failure. 
Attempting to delete a stack results in some zombie containers that the rancher UI shows as "Standalone" containers that are permanently in a "Stopping" state until you nuke the instance.

I saw that there's a PR in to make the ebs driver work nicely with m5 instances now, but it would also be good if there was some error code sent back so the user knows the driver can't find the block device to attach to -- would've saved me many hours :)

**Comment by jean-francois-labbe** [comment]
I'm facing this issue too

**Comment by marcusianlevine** [comment]
Just struggled with this for a couple of days, couldn't mount an EBS volume on an m5.large while it works perfectly well on an m4...

Don't see the PR mentioned above anywhere, was it merged?

**Comment by marcusianlevine** [comment]
Yeah looks like the underlying bug with EBS running on m5 instances was addressed at of 1.6.15: I just upgraded to 1.6.16 and the mount works now!

**Comment by loganhz** [comment]
Please let us know in a comment if you can still reproduce the issue, and we'll reopen it
